pages/community/mission.md

```diff
@@ -2,21 +2,17 @@
 layout: page
 title: Mission
 subtitle: Goals and Aims of the US-RSE Association
-bigimg:
 permalink: /mission/
 ---
 
-
 The US-RSE Association is centered around three main goals.  We aim
 to target our activities and actions to serving these three aspects:
```

Comment:
Since we've added a fourth bullet below, the two instances of "three" here need to change to "four."

Reply:
@cferenba Great catch! Updated with the "four" and removed the second reference, since it was redundant. "these aspects" should be adequate.